tests/contants.py

```diff
@@ -0,0 +1,25 @@
+import string
```

Comment:
The file name should probably read `constants.py`.

Reply:
I will move it.